### DISCUSSION TITLE
sonoff snzb-02 non espone dati

### DISCUSSION THREAD

**Discussion by dompa72** [indomus/forum#932] [HUB: Home Assistant]
Buonasera,
ho acquistato due Sonoff SNZB-02, ho associato i device in Zigbee2MQTT, purtroppo entrambi non espongono dati tranne linkquality. Come coordinator utilizzo CC2652P. Ho altri 18 dispostivi connessi tra cui 3 Aqara temperature, nessun problema.
Avevo in casa anche un Sonoff ZBMINI, provato l'associazione e questo non ha nessun problema.

ho letto anche questa discussione:
https://github.com/Koenkk/zigbee2mqtt/issues/8971

Anche cercando di riconfigurare il dispositivo, restituisce questo errore

Error Configure failed: Error: Bind 0x00124b00226ad2ff/1 msTemperatureMeasurement from '0x00124b0022ab79d8/1' failed (AREQ - ZDO - bindRsp after 10000ms)

Qualcuno con il mio stesso problema 
ha risolto?

**Comment by mkel84** [comment]
Salve, ho anche io lo stesso problema e non riesco a risolverlo.

**Reply by snakuzzo** [reply]
Dovete seguire la issue...qui possiamo fare ben poco.

**Comment by dompa72** [comment]
Con l'aggiornamento a me sono tornati a funzionare.
E' bastato ricollegarli, uno dei tre ho dovuto semplicemente cliccare su riconfigura e un piccolo click sul pulsante del sonoff
